build/dsc/templates/dsc_type.rb.erb

```diff
@@ -1,4 +1,5 @@
 require 'pathname'
+require File.expand_path(File.join(File.dirname(__FILE__), '../../puppet_x/puppetlabs/dsc_type_helpers'))
```

Comment:
For `require_relative`, just haven't used it so I don't know what issues it may have or if it would just work without issue.